apps/web/components/settings/DisableTwoFactorModal.tsx

```diff
@@ -12,7 +12,7 @@ import TwoFactorAuthAPI from "./TwoFactorAuthAPI";
 interface DisableTwoFactorAuthModalProps {
   open: boolean;
   onOpenChange: () => void;
-
+  disablePassword?: boolean;
```

Comment:
Needed a way to tell disable 2FA that the password should be asked, as when provider is not CAL, password is now correctly set to null